csunplugged/topics/management/commands/_CurriculumAreasLoader.py

```diff
@@ -1,47 +1,101 @@
 """Custom loader for loading curriculum areas."""
 
 import os.path
+
 from django.db import transaction
+
 from utils.BaseLoader import BaseLoader
+from utils.errors.MissingRequiredFieldError import MissingRequiredFieldError
+
 from topics.models import CurriculumArea
 
 
 class CurriculumAreasLoader(BaseLoader):
     """Loader for curriculum area content."""
 
-    def __init__(self, curriculum_areas_file, BASE_PATH):
+    def __init__(self, structure_file_path, BASE_PATH):
         """Create the loader for loading curriculum areas.
 
         Args:
             curriculum_areas_file: File path to YAML file (string)
```

Comment:
I see your point, but the new file name is more consistent across loaders and is more descriptive as it tells you it is a file *path*, not a loaded file. After working on the loaders for 2 weeks straight, I found this much easier to understand.